common/component/kafka/metadata.go

```diff
@@ -87,6 +88,12 @@ type KafkaMetadata struct {
 	channelBufferSize    int    `mapstructure:"-"`
 	consumerFetchMin     int32  `mapstructure:"-"`
 	consumerFetchDefault int32  `mapstructure:"-"`
+	// schema registry
+	SchemaRegistryURL       string        `mapstructure:"schemaRegistryURL"`
+	SchemaRegistryAPIKey    string        `mapstructure:"schemaRegistryAPIKey"`
+	SchemaRegistryAPISecret string        `mapstructure:"schemaRegistryAPISecret"`
+	SchemaCachingEnabled    bool          `mapstructure:"schemaCachingEnabled"`
+	LatestSchemaCacheTTL    time.Duration `mapstructure:"LatestSchemaCacheTTL"`
```

Comment:
I think I'd prefer if this started with "schema" for consistency with the other schema-related properties

```suggestion
	LatestSchemaCacheTTL    time.Duration `mapstructure:"latestSchemaCacheTTL"`
```